lib/hipe/cerl/erl_types.erl

```diff
@@ -198,6 +196,9 @@
 	 t_tuple_size/1, t_tuple_size/2,
 	 t_tuple_sizes/1,
 	 t_tuple_subtypes/1,
+         t_is_byte/1,
```

Comment:
Why were these functions moved?  All t_* functions were (supposed to, at least) sorted before, now they are not.

Reply:
Addressed in 11779fc.